### PR TITLE
Removes the temptation to re-add old R&D

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

**Comment by Buggy123** [comment]
No universal protolathe board fucks over free golems.

**Comment by uraniummeltdown** [comment]
old RnD was better than techwebs :middle_finger:

**Comment by Buggy123** [comment]
>old RnD was better than techwebs

**Comment by Buggy123** [comment]
Also its absurd to remove it from the code with the explicit intent to actively make it more difficult for anyone to re-add anything like this under any circumstance.

That's like if I added something, but also added a bunch of checks so that if anyone tried to remove it Byond won't compile. Like if you added old shittily coded ninjas but _intentionally_. Its hostile coding.

**Comment by Iamgoofball** [comment]
@Buggy123 
hexchat_2018-01-18_22-08-04
hexchat_2018-01-18_22-08-29

**Comment by Buggy123** [comment]
I stand by my words.

**Comment by 81Denton** [comment]
At least add the departmental lathe boards to /obj/item/storage/box/rndboards so that Free Golems don't get fucked over

**Comment by Buggy123** [comment]
Yep, this causes issues with free golems. Omnilathes are a core part of their functionality, so there aren't many options here: either this has to become "removes free golems: the PR", free golems have to get one of every lathe board(ew), or this PR is merged without altering free golems (causing a gamebreaking issue with them).

**Comment by arsserpentarium** [comment]
Goof,old RnD was shit. But lesser,than techwebs in present form.

**Comment by kevinz000** [comment]
@Iamgoofball it's one person being an issue all the time [awesine] don't remove it completely you're going to fuck over free golems and derelict old rnd unless you want to manually map in the new boards.

**Comment by GuyonBroadway** [comment]
Honestly at this rate the people against R&D are bitching and whining harder than the salty r&d mains they have a go at in every pr like this.

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		/obj/item/stock_parts/manipulator = 2,
 		/obj/item/stack/sheet/glass = 1)
 
-/obj/item/circuitboard/machine/abductor
+//obj/item/circuitboard/machine/protolathe/abductor
 	name = "alien board (Report This)"
 	icon_state = "abductor_mod"
```
Comment (deathride58) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: You aren't even commenting this out right

**Comment by Cyberboss** [comment]
Come on goof do it right

File: code/game/objects/items/circuitboards/machine_circuitboards.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm]: That's the git diff being really fucky. Check the actual file.

**Pull request closed without merging**